build/Foundation/dll/Foundation.def

```diff
@@ -874,4 +874,11 @@ LIBRARY Foundation
         ; MainDispatcher
         ScheduleMainRunLoop
         ScheduleMainRunLoopAsync
-        SetupMainRunLoopTimedMultipleWaiter
\ No newline at end of file
+        SetupMainRunLoopTimedMultipleWaiter
+
+        ;UTTypeHelpers (Internal)
+        _UTTypeCreatePreferredIdentifierForTag
+        _UTTypeCreateAllIdentifiersForTag
```

Comment:
The solution to the mixed up dependency chain is to remove the use of foundation (which, iirc, is *only logging-related*) from MobileCoreServices, not to merge MobileCoreServices into Foundation

---
In reply to: [95276171](https://github.com/Microsoft/WinObjC/pull/1662#discussion_r95276171) [](ancestors = 95276171)